tests/testthat/test-layer.r

```diff
@@ -38,6 +38,13 @@ test_that("missing aesthetics trigger informative error", {
   )
 })
 
+test_that("if an aes is mapped to a function that returns NULL, it is removed", {
+  df <- data.frame(x = 1:10)
+  wrap <- function(...) tryCatch(..., error = function(e) NULL)
```

Comment:
I think you can simplify this to just `function() NULL`

Reply:
Agreed.